sdk/storage/tests.yml

```diff
@@ -10,29 +10,30 @@ jobs:
             JavaVersion: '1.8'
             DisplayName: 'Windows - Java 8'
       ServiceDirectory: storage
+      TimeoutInMinutes: 120
       EnvVars:
         AZURE_TEST_MODE: LIVE
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

Comment:
Same comment as the AAD, are we wiring up our created test resources to allow the application represented by these secrets to have access via AAD?